src/golang/lib/collections/shared/engine.go

```diff
@@ -31,10 +33,19 @@ type AirflowConfig struct {
 	ArtifactMetadataPathPrefix map[uuid.UUID]string `json:"artifact_metadata_path_prefix"  yaml:"artifact_metadata_path_prefix"`
 }
 
+type K8sConfig struct {
+	IntegrationId uuid.UUID `json:"integration_id"  yaml:"integration_id"`
+}
+
 func (e *EngineConfig) Scan(value interface{}) error {
 	return utils.ScanJsonB(value, e)
 }
 
 func (e *EngineConfig) Value() (driver.Value, error) {
 	return utils.ValueJsonB(*e)
 }
+
+type K8sIntegrationConfig struct {
```

Comment:
Can you move this to `lib/collections/integration/config.go`. Maybe after #362 is merged